1-JavaScript/Week2/assignment/ex2-dogYears.js

```diff
@@ -16,15 +16,17 @@ calculate it!
    ages.
 -----------------------------------------------------------------------------*/
 
-function calculateDogAge(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function calculateDogAge(humanYearAge) {
+  const dogYearAge = humanYearAge * 7;
+  return `your doggie is ${dogYearAge} years old in dog years`;
```

Comment:
Please be more careful with the instructions: the tests compare every single character of the output, so it needs to match what has been asked.

```suggestion
  return `Your doggie is ${dogYearAge} years old in dog years!`;
```